Change sponsor details

BenGitsMagic opened the pull request.

ErikBoesen commented on _includes/footer.html:
```diff
@@ -1,6 +1,6 @@
 <footer>
     <h5>We love our sponsors!</h5>
-    <p>Please consider <a href="http://1418.team/sponsorship">sponsoring us</a>!</p>
+    <p style="color:white;">Please consider <a href="http://1418.team/sponsorship">sponsoring us</a>!</p>
```
Avoid inline styling. Color this with CSS instead.

BenGitsMagic replied:
How could I color just that line with CSS? When I use an internal style it changes all the paragraphs

ErikBoesen replied:
Add a class to the element (for example `.emphasized`) and then style that class.

ErikBoesen replied:
https://github.com/frc1418/frc1418.github.io/blob/master/css/main.sass#L377-L388

This is the relevant area you'll need to modify.

ErikBoesen reviewed (changes requested):
Looks good, just one small change!

ErikBoesen commented:
Also, make sure to pull from master and check out to a new branch in between PRs, otherwise your commits will be duplicated as seems to have happened between this PR and #502.

BenGitsMagic commented:
I changed the font change to be internal instead of inline, is this good now?

ErikBoesen commented:
This makes all the paragraphs in the footer white, when I think you only wanted to target the "sponsor us" text. You'd be best off making a class specifically for that emphasized text and then targeting `.yourclassname` instead of `p` in the SASS code.

BenGitsMagic commented:
I tried putting the class style in the SASS code and it gave me an error. Then I put it in a style at the top of the footer and it worked. Does this work, or is there a better way to do this in the SASS code?

ErikBoesen commented:
Can you send a screenshot or excerpt of how you tried to update the SASS? Putting `<style>` tags is also generally not good practice

BenGitsMagic commented:
I messed with it a little and I think I've done what you suggested now. I removed the <style> tag and added it to the style sheet. The issue before was just that I had brackets in the style sheet which I didn't know made it not work

ErikBoesen commented on _includes/footer.html:
```diff
@@ -1,6 +1,9 @@
     <h5>We love our sponsors!</h5>
-    <p>Please consider <a href="http://1418.team/sponsorship">sponsoring us</a>!</p>
+    <div class="emphasized">
+        <p>Please consider <a href="http://1418.team/sponsorship">sponsoring us</a>!</p>
```
Looks great! Last thing I'd say is that it would probably be better to just put this class on the `<p>` but otherwise this looks great, thanks!

BenGitsMagic commented:
Ok fixed that

ErikBoesen reviewed (approved):
Wonderful, thanks! Sorry for all the nitpicking :)

BenGitsMagic commented:
Thank you for all the help!

ErikBoesen commented:
Of course! Also, I would love to be added to the Slack group for this year or however y'all are keeping in touch. I graduated several years ago but would love to give programming advice wherever helpful.

BenGitsMagic commented:
Sure thing! We aren't on slack all that often but that would probably be the best way for us to contact you. What email do you want to have added?

ErikBoesen commented:
me@erikboesen.com would be great!

BenGitsMagic commented:
It actually looks like you're already in the slack group from what I can see

ErikBoesen commented:
Hmmm... what's the slack URL?

BenGitsMagic commented:
https://app.slack.com/client/T01AV0BH9TQ/C01B1MU6E2W is what it is on my computer, also in case it helps here is an invite link for the group: https://join.slack.com/t/gmhsroboticsteam-1418/shared_invite/zt-1gusvq8fa-QywkSwbAM8Wf_Lk1TAhMAg
